codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java

```diff
@@ -42,6 +44,9 @@ public HistoryAction(HistoryPresenter presenter,
     @Override
     public void actionPerformed(ActionEvent e) {
         eventLogger.log(this);
-        presenter.showDialog();
+        if (this.presenter == null) {
```

Comment:
Ah I hadn't seen that the HistoryPresenter was bound as a singleton.
So you're right, there's no need for that field.
Amending the PR now...